namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-community-accommodation-dev/06-certificates.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: hmpps-approved-premises-dev-cert
+  namespace: hmpps-approved-premises-dev
+spec:
+  secretName: hmpps-approved-premises-dev-cert
+  issuerRef:
+    name: letsencrypt-production
+    kind: ClusterIssuer
+  dnsNames:
+    - cas-approved-premises-dev.hmpps.service.justice.gov.uk
```

Comment:
@pezholio what do you reckon? This felt better than suffixing all three with `-2`

Reply:
Yeah, I'm happy with that for now